opus/application/static_media/js/browse.js

```diff
@@ -45,6 +45,12 @@ var o_browse = {
     tempHash: "",
     dataNotAvailable: false,
     onRenderData: false,
+
+    // font awesome icon class
+    pillSortUpArrow: "fas fa-arrow-circle-up",
```

Comment:
Since these will never change would it be better to somehow define them as "const"?